### ISSUE TITLE
Training set and test set problems

### ISSUE THREAD

**Issue by wangwei-bee** [MingjunZhong/seq2point-nilm#8]
I divided the training set and verification set according to the code you gave about 10:1, but if the two data are different, an error will be reported

**Comment by MingjunZhong** [comment]
I don't understand your question.
